ArjunaJTS/interop/glassfish/pom.xml

```diff
@@ -40,6 +40,7 @@
         <property>
           <name>!skipTests</name>
         </property>
+        <jdk>[1.5,10]</jdk>
```

Comment:
Does this work correctly with JDK9 and JDK10? Should not be there something like only `[1.5,8]`?
Plus, can you please add the xml comment there why this is skipped (some short info that the glassfish in the used version is not running well on jdk8/9/10+ or something)